src/components/ObjektDisplay.vue

```diff
@@ -0,0 +1,48 @@
+<script setup lang="ts">
+    const objektsQuery = `
+        query MyQuery 
+            {collections(limit: 18){
+                front
+                id
+            }
+        }
+        `;
+        
+    const response = await fetch("https://cosmo-api.gillespie.eu/graphql", {
+        method: "POST",
+        headers: {
+            "Content-Type": "application/json"
+        },
+        body: JSON.stringify({
+            query: objektsQuery,
+        })
+    });
+
+    const parsedData = await response.json();
+    const imageList = parsedData.data.collections
+
+
+
+    console.log(imageList);
+</script>
+
+<template>
+    <div id="title">
+        <h2>10 first objekts query</h2>
```

Comment:
Doesn't align with the query anymore, since that fetches 18.

Reply:
Was mostly used as a place holder to have a basic thing to display and know if the component was failing or me doing something wrong with the grid/query. I'll remove it.